### PR TITLE
Ziti api rename

### PR DESCRIPTION
**Pull request by ekoby**

### CODE REVIEW

File: library/ziti_src.c [hunk 1]
```diff
@@ -14,34 +14,41 @@ See the License for the specific language governing permissions and
...
 
 /**
  * Inherits from uv_lint_t and used to register as source link for `um_http`,
  * sening HTTP traffic over a Ziti connection.
  */
 typedef struct ziti_link_s {
     UV_LINK_FIELDS
-    nf_context nfc;
-    nf_connection conn;
+    ziti_context nfc;
```
Comment (dovholuknf) [inline library/ziti_src.c:27]: missed the variable name `nfc`

File: library/ziti.c [hunk 2]
```diff
@@ -622,21 +613,21 @@ static const ziti_version sdk_version = {
...
 };
 
-const ziti_version *NF_get_version() {
+const ziti_version *ziti_get_version() {
     return &sdk_version;
 }
 
 static void grim_reaper(uv_prepare_t *p) {
-    nf_context nf = p->data;
+    ziti_context nf = p->data;
```
Comment (dovholuknf) [inline library/ziti.c:621]: another variable that's used in so many places... maybe you left it on purpose? :)

File: library/ziti.c [hunk 3]
```diff
@@ -49,10 +48,10 @@ static const char *ALL_CONFIG_TYPES[] = {
 };
 
 struct nf_init_req {
-    nf_context nf;
+    ziti_context nf;
```
Comment (dovholuknf) [inline library/ziti.c:51]: so many places i referred to the wrong one :)

File: library/internal_model.c [hunk 4]
```diff
@@ -16,8 +16,8 @@ limitations under the License.
 
 
 #include <internal_model.h>
-#include <nf/ziti_model.h>
-#include <nf/errors.h>
+#include <ziti/ziti_model.h>
+#include <ziti/errors.h>
 
 #include <string.h>
```
Comment (dovholuknf) [inline library/internal_model.c:23]: github won't let me comment on `IMPL_MODEL(nf_id` line 28

File: library/ziti_src.c (hunk 1 above)
Comment (ekoby) [reply library/ziti_src.c:27]: only focused on external stuff (API, samples)

File: library/connect.c [hunk 5]
```diff
@@ -26,13 +26,13 @@ static const char *TYPE_DIAL = "Dial";
 #define crypto(func) crypto_secretstream_xchacha20poly1305_##func
 
 struct nf_conn_req {
-    struct nf_conn *conn;
```
Comment (dovholuknf) [inline library/connect.c:29]: struct name is `nf_conn_req`

File: library/ziti_src.c (hunk 1 above)
Comment (dovholuknf) [reply library/ziti_src.c:27]: in the samples search for `nf_` and you'll find about 20 more

File: includes/ziti/ziti_src.h [hunk 6]
```diff
@@ -25,7 +25,7 @@ limitations under the License.
 #define NF_ZITI_SRC_H
```
Comment (dovholuknf) [inline includes/ziti/ziti_src.h:25]: defining NF_ZITI_SRC_H

File: library/ziti_src.c (hunk 1 above)
Comment (dovholuknf) [reply library/ziti_src.c:27]: actually, try `grep -ri nf programs| grep -v config | grep --color=auto nf` found 35 total locally

File: library/ziti.c (hunk 2 above)
Comment (ekoby) [reply library/ziti.c:621]: not chasing individual variables

**Pull request merged**